modules/saml/docs/sp.md

```diff
@@ -219,6 +219,9 @@ Options
     in the IdP-remote metadata overrides this the option in the SP
     configuration.
 
+`disable_unsolicited`
+:    Whether this SP will refuse to process unsolicited responses. The default value is `false`.
+
```

Comment:
I personally find it better understandable if the toggle is named "enable_unsolicited", defaulting to  "true" over "disable=false".